### PR TITLE
Integrate posthog and log events to track usage

### PR DESCRIPTION
**Pull request by ykeremy**

### CODE REVIEW

**Review by wintonzheng — APPROVED** [review]

File: scripts/tracking.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+from skyvern.forge.sdk.settings_manager import SettingsManager
+
+posthog = Posthog(
+    "phc_bVT2ugnZhMHRWqMvSRHPdeTjaPxQqT3QSsI3r5FlQR5",
+    host="https://app.posthog.com",
+    disable_geoip=False,
+)
+
+DISTINCT_ID = "oss"
```
Comment (ykeremy) [inline scripts/tracking.py:17]: will be updated in another PR
